ftl-build/test_build.sh

```diff
@@ -1,12 +1,15 @@
 #!/bin/bash
 
-# exit when any command fails
-set -e
-
 # Walk subdirectories
 for dir in ./*/
 do
         pushd "$dir"
-        docker build .
+        if ! docker build .; then
```

Comment:
Not a request for immediate change.. just wondering if there is a way to make this (optionally) asynchronous